go.mod

```diff
@@ -3,10 +3,12 @@ module github.com/LUSHDigital/core
 go 1.14
 
 require (
+	github.com/LUSHDigital/core-lush v0.3.1
 	github.com/LUSHDigital/uuid v0.2.0
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/dustin/go-humanize v1.0.0
 	github.com/golang/protobuf v1.3.2
+	github.com/gorilla/mux v1.7.4
```

Comment:
Why are we importing this?

Reply:
I don't even understand why these changes are here, let me go mod tidy and push again

Reply:
pushed new patch to remove this